lisa/util/process.py

```diff
@@ -411,6 +435,30 @@ def _check_if_need_input_password(self, raw_input: str) -> None:
                 " which is not support in Lisa now"
             )
 
+    def _filter_sudo_required_password_info(self, raw_input: str) -> str:
```

Comment:
As mentioned in other comments, don't assume the content is the same. Detect it with "sudo echo OK".

Reply:
Only the first check output in suport_sudo is "sudo echo OK". Why detect it with "sudo echo OK"?

Reply:
I mean, when use "sudo echo OK". the password prompt should be saved, because the prompt can be changed, or localized.

Reply:
Done